.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         pip install -r dev_requirements.txt
     - name: Lint with Pylint
       run: |
-        pylint src
+        pylint msgraphcore
```

Comment:
We should skip running this when the path is not msgraphcore or tests

Reply:
Also, if there is ever a reason to skip ci like what we have in ADO, you can add this: https://github.com/microsoftgraph/msgraph-typescript-typings/pull/130#pullrequestreview-389521771

Reply:
This runs the linter. We nee to run the linter against msgraphcore